### PR TITLE
Run Ansible lint

### PR DESCRIPTION
**Pull request by ccamacho**
First patch to run lint checks in the project

### CODE REVIEW

File: toolbox/build/venv-requirements.txt [hunk 1]
```diff
@@ -8,3 +8,6 @@ pytest-xdist>=1.30.0
 pycodestyle>=2.5.0
 voluptuous>=0.11.7
 yamllint>=1.20.0
+flake8==3.7.7
+ansible-lint==4.1.0
+bashate
```
Comment (jistr) [inline toolbox/build/venv-requirements.txt]: Nit: worth keeping alphabetic order here? Should we specify minimum version for bashate?

File: Makefile [hunk 2]
```diff
@@ -33,6 +33,13 @@ os_migrate-os_migrate-latest.tar.gz:
 
 # TESTS
 
+test-lint:
```
Comment (jistr) [inline Makefile]: We should add it to test-fast target like: `test-fast: test-lint test-sanity test-unit`. The idea of it was "all checks which don't need OpenStack running".

File: .github/workflows/linters.yml [hunk 3]
```diff
@@ -0,0 +1,34 @@
+---
+name: linters
```
Comment (jistr) [inline .github/workflows/linters.yml]: I'm wondering if it's worth another job or if the unit test job should simply run the `test-fast` target and test all lints + unit tests. The setup time (building container etc.) is probably comparable or longer to the time of actually executing linters + unit tests? I guess i'd prefer just one job for all "blocking checks which do not require running an OpenStack env, and are therefore fast to execute", but it's not a strong opinion, i'd be fine with separate jobs too in case we collectively feel it's a better way.

Comment (ccamacho) [reply .github/workflows/linters.yml]: Im checking the actual difference between jobs and workflows, and it seems the workflows are totally independent.

What we can use is to define 2 jobs, one for consistency checks and then the functional tests execution which depends on the consistency check passing.

File: toolbox/build/venv-requirements.txt (hunk 1 above)
Comment (ccamacho) [reply toolbox/build/venv-requirements.txt]: yeahp, exporting the requirements usually adds the version and sorted alphabetically.

File: Makefile (hunk 2 above)
Comment (ccamacho) [reply Makefile]: yeahp, done.

File: scripts/linters.sh [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+#
+# Flake
+# TODO: Have the Ansible and YAML lint fixed first
+#flake8 --exclude releasenotes,.tox --ignore E125,E251,E402,H405,W503,W504
+
+# Bash hate
+#find ./ -not -wholename "*.tox/*" -and -not -wholename "*.test/*" -and -name "*.sh" -print0 | xargs -0 bashate -v --ignore E006
+
+#Yaml lint
+#find ./ -not -wholename "*.tox/*" -and -name "*.yml"  -print0 | xargs -0 yamllint
```
Comment (jistr) [inline scripts/linters.sh]: Ack i assume this will be enabled in another PR, LGTM.

**Comment by jistr** [comment]
I looked at CI logs too, they seem good.
